test/database_test.go

```diff
@@ -162,13 +117,10 @@ func TestDatabase_UserDelete(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	sub := "this is a sub"
```

Comment:
(nit) use an actual `sub` here, that way the reader knows how it looks and has a better feel for what it is

Reply:
Done.